src/screens/Profile/Home/index.tsx

```diff
@@ -269,6 +269,22 @@ export default function ProfileHome({navigation}: ProfileScreenProps) {
               </View>
             </View>
           </Sections.GenericItem>
+          <Sections.ActionItem
+            mode="toggle"
+            text={t(ProfileTexts.sections.newFeatures.frontPage)}
+            checked={newFrontPage}
+            testID="newFrontpageToggle"
+            onPress={(newFrontPage) => {
+              analytics().logEvent('toggle_beta_frontPage', {
+                toggle: newFrontPage ? 'enable' : 'disable',
+              });
+              updateMetadata({
+                'AtB-Beta-Frontpage': newFrontPage ? 'enabled' : 'disabled',
+              });
+              setPreference({newFrontPage});
+            }}
+          />
+
```

Comment:
We should put this under "Development menu" instead of "New features" so it's not visible to end users when we do a release. We can move it to here once its ready for beta testing.